CONTRIBUTING.md

```diff
@@ -33,16 +37,32 @@ $ git clone git@github.com:<user>/kubebuilder.git $GOPATH/src/sigs.k8s.io/kubebu
 1. Build the project by using the command `make build` 
 1. Run the command `make install` to create a bin with the source code 
 
-## How to test kubebuilder locally
+**NOTE** It will allow you to test the changes made since when you run `kubebuilder` commands, it will use the binary built from your source code. Run `which kubebuilder` to check its path.
```

Comment:
I don't think this line adds anything to the previous content.

Reply:
These lines make clear for new users how to test the changes locally, that the make install will gen the bin based on the source code.

Reply:
It is not clear for new contributors. The idea is to make clear that it allow them test their changes locally

Reply:
This section is called `How to build kubebuilder locally` and the next one was called `How to test kubebuilder locally`. With that info, do you still need to be told that if you build locally you will use the local binary?

Reply:
It is not clear for all dev. Many of them raising questions to understand that the make install will install the bin from the source code.

Reply:
@Adirio, 

However, I changed it to address your concerns and I am closing this one. 
Please, let me know wdyt now.